content/docs/command-reference/exp/run.md

```diff
@@ -179,28 +194,37 @@ experiment we just ran (`exp-44136`).
 
 ## Example: Modify parameters on-the-fly
 
-`dvc exp run--set-param` (`-S`) saves you the need to manually edit the params
-file before running an experiment.
+`dvc exp run --set-param` (`-S`) saves you the need to manually edit the params
+file before running an experiment. It can override (`train.epochs=10`), append
+(`+train.weight_decay=0.01`), or remove (`~model.dropout`) parameters.
+
+<admon type="tip">
+
+DVC supports the
+[Hydra Override Grammar](https://hydra.cc/docs/advanced/override_grammar/basic/),
+with the exception of patterns that modify Hydra's defaults list.
+
+</admon>
```

Comment:
All of this info should probably be somewhere in the **Description**. An Example is too specific to be the only place where general behavior is described.